sources/updater/updog/src/bin/updata.rs

```diff
@@ -142,25 +142,45 @@ struct WaveArgs {
     arch: String,
 
     // start bound id for this wave (0 <= x < 2048)
-    #[structopt(short = "b", long = "bound-id")]
-    bound: u32,
+    #[structopt(
+        short = "b",
+        long = "bound-id",
+        requires = "start",
+        conflicts_with = "wave_file"
```

Comment:
Why not just require a wave file to use updata?  Specifying waves was confusing and awkward anyway; it's easier to write a bit of TOML.  And it's safer to fully replace waves instead of adding/removing - they have to be compatible with each other.  And it'd get rid of any difference between how to specify individual vs. all.  And we'd get rid of the similarly dangerous remove operations and simplify the tool further.